portal/server/controllers/dashboard/deals/index.js

```diff
@@ -29,7 +29,6 @@ const getAllDealsData = async (
   const filtersArray = submittedFiltersArray(sessionFilters);
 
   const filtersQuery = dashboardDealsFiltersQuery(
-    sessionFilters.createdByYou,
```

Comment:
are we not using the createdByYou filter from the session?

Reply:
no.  Using normal filter logic now so can be done in dashboardDealsFiltersQuery